bitnami/spring-cloud-dataflow/templates/server/configmap.yaml

```diff
@@ -45,12 +45,16 @@ data:
                     {{- if $environmentVariables }}
                     environmentVariables: {{- $environmentVariables | trim | nindent 22 }}
                     {{- end }}
+                    {{- if .Values.deployer.resources }}
                     {{- if .Values.deployer.resources.limits }}
                     limits: {{- include "common.tplvalues.render" (dict "value" .Values.deployer.resources.limits "context" $) | trim | nindent 22 }}
                     {{- end }}
                     {{- if .Values.deployer.resources.requests }}
                     requests: {{- include "common.tplvalues.render" (dict "value" .Values.deployer.resources.requests "context" $) | trim | nindent 22 }}
                     {{- end }}
+                    {{- else if ne .Values.deployer.resourcesPreset "none" }}
+                    {{- include "common.resources.preset" (dict "type" .Values.deployer.resourcesPreset) | nindent 20 }}
+                    {{- end }}
```

Comment:
This syntax does not require the `resources:` property, right? It is defining limits/requests directly:

```yaml
data:
  application.yaml: |-
    spring:
      cloud:
        dataflow:
          task:
            platform:
              kubernetes:
                accounts:
                  default:
                    limits:
                      cpu: 150m
                      memory: 192Mi
                    requests:
                      cpu: 100m
                      memory: 128Mi
```

Reply:
Correct